config.py

```diff
@@ -5,12 +5,20 @@ class Config(object):
     TESTING = False
 
     SECRET_KEY = secrets.token_hex(16)
+    SQLALCHEMY_ECHO = False
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+    DEBUG_TB_INTERCEPT_REDIRECTS = False
+    SQLALCHEMY_DATABASE_URI = 'postgresql:///gamr'
```

Comment:
Add a parent configuration class from which these settings can be inherited by other configuration classes.